src/components/containers/MetricContainer.js

```diff
@@ -34,9 +34,10 @@ class MetricContainer extends React.Component {
                     xUnit={metricFetch.value.units ? metricFetch.value.units.x : undefined}
                     yUnit={metricFetch.value.units ? metricFetch.value.units.y : undefined}
                     chartDataURL={metricFetch.meta.request.url}
+                    showOutliers={this.props.showOutliers}
 
                     // Force to "false" if undefined
-                    asOverlay={this.props.asOverlay === true}
+                    asOverlay={!!this.props.asOverlay}
```

Comment:
This changes the logic a bit. The original sets `this.props.asOverlay` to `false` if it's not a boolean. In the new line, if `this.props.asOverlay` isn't a boolean, it will be coerced to one. E.g., `!!'false'` would become `true`.

Reply:
Great example, this won't work with strings.